classes/PsCheckoutCart.php

```diff
@@ -30,6 +30,12 @@ class PsCheckoutCart extends ObjectModel
     const STATUS_PENDING_APPROVAL = 'PENDING_APPROVAL';
     const STATUS_PARTIALLY_COMPLETED = 'PARTIALLY_COMPLETED';
 
+    const PAYPAL_ENVIRONMENT_PROD_LIVE = 'PROD_LIVE';
+    const PAYPAL_ENVIRONMENT_PROD_SANDBOX = 'PROD_SANDBOX';
+    const PAYPAL_ENVIRONMENT_PREPROD_LIVE = 'PREPROD_LIVE';
+    const PAYPAL_ENVIRONMENT_PREPROD_SANDBOX = 'PREPROD_SANDBOX';
```

Comment:
You can remove this and use https://github.com/PrestaShopCorp/ps_checkout/blob/v8.3.3.1/src/PayPal/Mode.php for now cause the others environments are not supported yet so we will have only `SANDBOX` & `LIVE`